src/integrations/codeSnippets.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+import yaml from 'js-yaml';
+import fs from 'fs';
+import path from 'path';
+
+function transformYamlToHAR(data) {
```

Comment:
@MeganSteenkamp Suggestion: Keep this code as a standalone module and wrap it in an Astro integration. Makes testing less convoluted